### PR TITLE
Add a CLI argument to disable active clipboard stealing

### PR DESCRIPTION
**Pull request by Res260**
uses only passive stealing instead

- Add pyrdp-generated files to .gitignore

Tested using mstsc.exe on win 10 to a win7 target machine

### CODE REVIEW

File: bin/pyrdp-mitm.py [hunk 1]
```diff
@@ -1,20 +1,21 @@
 #!/usr/bin/env python3
 
+import argparse
```
Comment (alxbl) [inline bin/pyrdp-mitm.py]: Should go below license?

Comment (Res260) [reply bin/pyrdp-mitm.py]: fixed

**Comment by Res260** [comment]
should be good now @alxbl

**Comment by Res260** [comment]
nvm

**Comment by Res260** [comment]
ok now it should be good lol

**Pull request merged**